### PR TITLE
Homepage cards

### PR DESCRIPTION
**Pull request by timcowlishaw**
## Changes in this PR:

New layout for homepage, which also links the previously deployed new courts and tribunals page.

`cards_homepage` feature flag will need to be set for this to be visible.

## Trello card / Rollbar error (etc)
https://trello.com/c/YlzzrDFX/1707-%F0%9F%8F%A1-homepage-changes-3-cards-screen
## Screenshots of UI changes:
Screenshot 2024-03-13 at 11 14 38
Screenshot 2024-03-13 at 11 15 01

### CODE REVIEW

**Comment by timcowlishaw** [comment]
[🏡 Homepage changes - '3 cards' screen](https://trello.com/c/YlzzrDFX/1707-%F0%9F%8F%A1-homepage-changes-3-cards-screen)

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   ],
   "dependencies": {
     "@nationalarchives/ds-caselaw-frontend": "github:nationalarchives/ds-caselaw-frontend#v2.0.1",
+    "@nationalarchives/frontend": "^0.1.41",
```
Comment (timcowlishaw) [inline package.json:39]: Oh, I need to get rid of this!

**Review by dragon-dxw — APPROVED** [review]
Are we definately happy with the picture choices / licences, mostly the judgment that's being headlined?

Are the pictures too big on mobile?

File: ds_judgements_public_ui/static/images/browse-courts-tribunals.png
Comment (CristinaRO) [inline ds_judgements_public_ui/static/images/browse-courts-tribunals.png:1]: 800KB is pretty large for a decorative image, especially for people who might be browsing on mobile data...

File: ds_judgements_public_ui/templates/includes/cards_homepage.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% load static %}
+<ul class="cards-homepage">
+  <li>
+    {% include "includes/card_homepage.html" with title="Recently published" body="Browse legal decisions most recently added to the Find Case Law site." link_route="search" image_src="/images/recently_added.png" %}
+  </li>
+  <li>
+    {% include "includes/card_homepage.html" with title="Browse by court or tribunal" body="Browse judgments and tribunals from the United Kingdoms's 18 courts and 7 tribunals." link_route="courts_and_tribunals" image_src="/images/browse-courts-tribunals.png" %}
+  </li>
+  <li>
+    {% include "includes/card_homepage.html" with title="Help and Support" body="Find out how to search for judgements and decisions." link_route="how_to_use_this_service" image_src="/images/help-and-guidance.png" %}
```
Comment (CristinaRO) [inline ds_judgements_public_ui/templates/includes/cards_homepage.html:10]:
```suggestion
    {% include "includes/card_homepage.html" with title="Help and Support" body="Find out how to search for judgments and decisions." link_route="how_to_use_this_service" image_src="/images/help-and-guidance.png" %}
```

**Pull request merged**